### DISCUSSION TITLE
Make the title of Table of Contents do what the icon does - expand/collapse

### DISCUSSION THREAD

**Discussion by MariuszMilka** [elementor/elementor#29713] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I am always frustrated when I set up the Table of Contents widget. All functionality seems to be there apart from expanding/collapsing the list when the title is clicked. Instead I need to click the small icon on the lefft. Normally on Desktop I keep the list expanded, so not a big problem. But on tablet I have it collapsed by default. Then to expand the list, I need to click a small icon instead the whole heading or the arrow or the title. In accordion widgets clicking in the header nornally expands it. But in the Table of Contents, which technically is a single accordion item, only the icon expands/collapses the list.

### Describe the solution you'd like.

It would be nice If I could click in the Table of Contents tilte of the whole header and be able to expand/collapse it.

### Describe alternatives you've considered.

There is no alternative.

### Additional context

Here is a similar topic, but noe one has ever responded to it: https://github.com/orgs/elementor/discussions/23835

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by sara-kariv** [comment]
+++++++1
